docs/en/observability/synthetics-projects-workflow.asciidoc

```diff
@@ -0,0 +1,84 @@
+
+++++
+<titleabbrev>Working with project monitors</titleabbrev>
+++++
+
+beta[]
+
+Projects are the most powerful and sophisticated way to configure synthetic monitors in the Elastic Stack. Projects let you define your infrastructure as code, more commonly known as IaaC or git-ops. With project monitors you organize your YAML configuration and Javascript/Typescript defined monitors on the filesystem, use Git for version control, and deploy via a CLI tool, usually executed on a CI/CD platform.
+
+
+== Getting started
+
+Working with projects requires working with the Elastic Synthetics CLI tool, which
+can be invoked via the `npx @elastic/synthetics` command. Before getting started
+you'll need to install Node.js; the `npx` command is provided by node, downloading
+the `@elastic/synthetics` NPM module if not yet available on your system.
+
+Run `npx @elastic/synthetics -h` to display the help menu and confirm your
+system is setup correctly
+
+It is highly recommended to use an https://cloud.elastic.co/[Elastic Cloud] deployment with this tutorial.
+If you wish to use a self-managed deployment, please read the <<synthetics-private-location>> docs first, and setup a private location with Fleet before proceeding.
+
+We'll start by creating our first project, run the command below to create a new 
+project named `projects-test` in the current directory.
+
+`npx @elastic/synthetics init projects-test`
+
+Then, follow the prompts on screen to setup the correct default variables for your project.
+When complete, set the `SYNTHETICS_API_KEY` environment variable in your terminal, which is used
+for authentication with your Elastic Stack. You will most likely want to set this permanently. 
+This is done differently for https://learn.microsoft.com/en-us/powershell/module/microsoft.powershell.core/about/about_environment_variables?view=powershell-7.2#saving-changes-to-environment-variables[Powershell] and https://unix.stackexchange.com/a/117470[Bash].
+
+Let's pause for a moment and examine the key files and directories inside your project. 
+
+* `journeys` contains the `.yaml`, `.ts` and `.js` files defining your monitors.
+* `synthetics.config.ts` contains settings for your project
+* `package.json` contains NPM settings for your project. Learn more in the https://docs.npmjs.com/about-packages-and-modules[NPM documentation].
+* `.github` contains sample Github actions workflow files for use with Github Actions.
+
+While inside our project directory we can do two things with the `npx @elastic/synthetics` command:
+
+* Run browser based monitors defined in `.ts` and `.js` files)
+* Push configurations to 
+
+Let's try running the example journeys in the `journeys` folder by executing the following command.
+
+`npx @elastic/synthetics journeys`
+
+Will run the browser journeys locally.
+
+Next, let's try pushing our journeys to our Elastic deployment by running the following command
+from inside your project.
+
+`npx @elastic/synthetics push`
```

Comment:
Use second person. 

Also, I think this is a bit repetitive. We could condense the list and the code blocks:

```suggestion
While inside the project directory you can do two things with the `npx @elastic/synthetics` command:

* Test browser-based monitors locally. To run all journeys defined in `.ts` and `.js` files:
+
[source,sh]
----
npx @elastic/synthetics journeys
----
* Push all monitor configurations to an Elastic deployment. Run the following command from inside your project:
+
[source,sh]
----
npx @elastic/synthetics push
----
```

Looks like:

<img width="860" alt="Screen Shot 2022-11-09 at 3 12 34 PM" src="https://user-images.githubusercontent.com/10479155/200942856-00b09c42-4951-4b0b-b8f8-a606c30f38d0.png">

Reply:
Much better